src/libs/OptionsListUtils.ts

```diff
@@ -1306,15 +1287,14 @@ function getOptions(
     let userToInvite: ReportUtils.OptionData | null = null;
     if (
         canCreateOptimisticPersonalDetailOption({
-            searchValue,
             recentReportOptions,
             personalDetailsOptions,
             currentUserOption,
             excludeUnknownUsers,
         })
     ) {
         userToInvite = getUserToInviteOption({
-            searchValue,
+            searchValue: '',
```

Comment:
Good catch, thats interesting, because that means that this is probably also an unused code path. (Almost) all code paths before were just passing `searchValue: ''`, making it early return as well.
Will carefully double check to see if it can be removed / needs to be treated differently

Reply:
`userToInvite` is only generated from a user's search input. As we removed the `searchValue` from `getOptions` we don't need to keep the code for `userToInvite` there.

Please check this commit https://github.com/Expensify/App/pull/53056/commits/d3ef36e701bcca7c6079fb7af84e9da6bb5042c6

Seems to work (but would appreciate a careful double check from your side):

![CleanShot 2024-11-27 at 12 22 03](https://github.com/user-attachments/assets/f6b6dd66-f5bb-4619-be95-cc516aca61fc)